src/main/java/com/clussmanproductions/trafficcontrol/blocks/BlockStand.java

```diff
@@ -0,0 +1,85 @@
+package com.clussmanproductions.trafficcontrol.blocks;
+
+import com.clussmanproductions.trafficcontrol.ModTrafficControl;
+import com.clussmanproductions.trafficcontrol.util.CustomAngleCalculator;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.properties.PropertyInteger;
+import net.minecraft.block.state.BlockStateContainer;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.renderer.block.model.ModelResourceLocation;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.item.Item;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+import net.minecraftforge.client.model.ModelLoader;
+
+public class BlockStand extends Block {
+	public static PropertyInteger ROTATION = PropertyInteger.create("rotation", 0, 15);
+	public BlockStand()
+	{
+		super(Material.IRON);
+		setRegistryName("stand");
+		setUnlocalizedName(ModTrafficControl.MODID + ".stand");
+		setCreativeTab(ModTrafficControl.CREATIVE_TAB);
+		setLightOpacity(1);
```

Comment:
Probably should add some kind of hardness values and harvest level and stuff. See other blocks for examples (as long as they have them, I know I'm missing that in a couple spots)

Reply:
this is done on purpuse as its ment to be easily moved as a sign stand would be irl